ios-push-app/Base.lproj/LaunchScreen.storyboard

```diff
@@ -1,22 +1,20 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
We actually need to keep the module name as `iosapp`. The reason is that our code generator does not know about the specific skit name and expects only `iosapp`. We are making a new generation of code generator that will be better, but for now we are avoiding making major changes to the existing one.